faq/README.md

```diff
@@ -175,22 +177,18 @@ Next to that label, it will tell you the plugin version.
 
 ### Where are the log files saved?
 
-Go to Plugins > FFXIV Settings and look for the `Log file location`.
+Go to Plugins > FFXIV Settings and look for the `Log file location`. 
+Or, search for `%appdata%\Advanced Combat Tracker\FFXIVLogs` in your start menu.
```

Comment:
This addition is incorrect.  The reason this FAQ suggests to look in the settings is because it could be pointed somewhere else.

Reply:
Changed it